app/Console/Commands/MakeDefaultURLsNull.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Illuminate\Console\Command;
+
+class MakeDefaultURLsNull extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:nullurl';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Change "default" URLs in the Posts table to null';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        info('rogue:nullurl: Starting!');
+
+        // Create the progress bar
+        $bar = $this->output->createProgressBar(Post::where('url', 'default')->count());
```

Comment:
If you want, you could extract the query to a variable and use it for both the count & iterator:

```php
$query = Post::where('url', 'default');
$bar = $this->output->createProgressBar($query->count());

// Get all posts with "default" for the url
$query->chunkById(100, function ($posts) use ($bar) {
   // ...
```

Reply:
YES I didn't like doing it twice felt un-DRY. did not realize my dreams could come true 🌠